contributing_to_docs/doc_guidelines.adoc

```diff
@@ -725,16 +725,17 @@ a|Use the following command to roll back a Deployment, specifying the Deployment
 `oc rollback <deployment>`
 
 |Use single asterisks for web console / GUI items (menus, buttons, page titles, etc.).
+Use two characters to form the arrow in a series of menu items, $$->.
 
 a|Choose $$*Cluster Console*$$ from the list.
 
-Navigate to the $$*Operators -> Catalog Sources*$$ page.
+Navigate to the $$*Operators* $$-> $$*Catalog Sources*$$ page.
```

Comment:
Actually, adding the $$'s in the middle there is causing the arrow to render. You should only need the $$'s surrounding the entire thing:

```
"$$*Operators* -> *Catalog Sources*$$"
```